### PR TITLE
restructure dirs

### PR DESCRIPTION
**Pull request by pld**

### CODE REVIEW

File: docs/engineering/android-app/developer-setup/testing/readme.mdx [hunk 1]
```diff
@@ -0,0 +1 @@
+# Testing
```
Comment (pld) [inline docs/engineering/android-app/developer-setup/testing/readme.mdx:1]: add generic testing content

**Comment by dubdabasoduba** [comment]
@pld Does this mean we will move the current testing documentation to the new generic category?

**Comment by pld** [comment]
No current test docs are about unit tests, generic/overall testing content goes in the new file

File: docs/engineering/android-app/developer-setup/testing/readme.mdx (hunk 1 above)
Comment (pld) [reply docs/engineering/android-app/developer-setup/testing/readme.mdx:1]: @Mstjamush I have assigned this PR to you can you please add some notes on integration testing, other quality assurance / forms of testing we do here. Let me know when that's done then I can do a review and merge, thank you

cc @AngelaKabari

**Comment by pld** [comment]
replaced by https://github.com/opensrp/fhircore/pull/2928/files

**Pull request closed without merging**